app/routes/__docs/$/$.tsx

```diff
@@ -71,11 +71,14 @@ export const loader: LoaderFunction = async ({ params, request }) => {
 
 export const meta: MetaFunction = ({ data, location }) => {
   const typedData = data as LoaderData
+  const title = typedData.page.frontmatter?.title
+  const description = typedData.page.frontmatter?.description || ""
   if (typedData && typedData.page) {
```

Comment:
If typedData.page is undefined there would be an exception on line 74

Reply:
Ok will address this.